src/types/onyx/DismissedReferralBanners.ts

```diff
@@ -0,0 +1,11 @@
+import type CONST from '@src/CONST';
+
+type DismissedReferralBanners = {
+    [CONST.REFERRAL_PROGRAM.CONTENT_TYPES.MONEY_REQUEST]?: boolean;
+    [CONST.REFERRAL_PROGRAM.CONTENT_TYPES.START_CHAT]?: boolean;
+    [CONST.REFERRAL_PROGRAM.CONTENT_TYPES.SEND_MONEY]?: boolean;
+    [CONST.REFERRAL_PROGRAM.CONTENT_TYPES.REFER_FRIEND]?: boolean;
+    [CONST.REFERRAL_PROGRAM.CONTENT_TYPES.SHARE_CODE]?: boolean;
+};
+
+export default DismissedReferralBanners;
```

Comment:
Could you update the description to also include the background for these changes for this dismissedReferralBanner nvp?

Reply:
I don't follow, what background? 😕

Reply:
The explanation for why the changes to this nvp, it sees like it was refactored more than others in this PR